infra/base-images/base-runner/bad_build_check

```diff
@@ -62,78 +62,69 @@ function check_startup_crash {
     CHECK_PASSED=1
   fi
 
-  if (( $CHECK_PASSED == 0 )); then
-    echo "BAD BUILD: the fuzzer seems to have either startup crash or exit."
-    exit 1
+  if [ "$CHECK_PASSED" -eq "0" ]; then
+    echo "BAD BUILD: the fuzzer seems to have either startup crash or exit." >> $LOG_PATH_FOR_BROKEN_TARGET
```

Comment:
Why do ">> $LOG_PATH_FOR_BROKEN_TARGET" throughout file, instead why not redirect output from running this script to LOG_PATH_FOR_BROKEN_TARGET

Reply:
That's a good idea. The only complication would be that I'll have to delete that file if all the checks have been passed, but that would be still better than redirecting everywhere. Done, thanks!

Reply:
That actually simplified the script a lot, thanks again!